analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Monitor.Conditions.CSharp11.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
```

Comment:
Education: why is this test really needed? Does the SE care if the method is in an interface or a class?

Reply:
It does not care, however it is not bad in my opinion to document it that it does not.